plugins/woocommerce/src/Internal/Admin/WCPayPromotion/WCPaymentGatewayPreInstallWCPayPromotion.php

```diff
@@ -24,6 +26,10 @@ class WCPaymentGatewayPreInstallWCPayPromotion extends \WC_Payment_Gateway {
 	 * Constructor
 	 */
 	public function __construct() {
+		if ( Features::is_enabled( 'reactify-classic-payments-settings' ) ) {
```

Comment:
I guess this is because we use the new suggestions logic if the new settings are enabled? Can we add a comment for posterity?

Reply:
Ok. Adding a comment.